### PR TITLE
Update to 1.16

### PR DESCRIPTION
**Pull request by simonbray**
The autobuilder script made a few strange changes, especially adding a lot of `module.py` files, which I reverted. Overall it was a very smooth process though.

### CODE REVIEW

File: parsec/commands/datasets/get_datasets.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import click
 from parsec.cli import pass_context, json_loads
-from parsec.decorators import custom_exception, list_output
+from parsec.decorators import custom_exception, dict_output
```
Comment (hexylena) [inline parsec/commands/datasets/get_datasets.py]: hmm, this looks wrong, I wonder what happened here.
it's definitely list[dict]
https://bioblend.readthedocs.io/en/latest/api_docs/galaxy/all.html?highlight=get_datasets#bioblend.galaxy.datasets.DatasetClient.get_datasets

**Comment by hexylena** [comment]
Thanks for doing this @simonbray! This looks good overall, but clearly need to figure out what went wrong with the output type detection. I'll look into it.

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Fantastic. Thanks for doing the update @simonbray!

**Comment by hexylena** [comment]
It's live. Also migrated it to github actions since travis is travis.

**Comment by simonbray** [comment]
Nice. Thanks a lot @hexylena!

**Pull request merged**